src/ui/src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -141,6 +215,12 @@ const useDialogStyles = makeStyles((theme: Theme) => createStyles({
   },
 }), { name: 'DialogDropdown' });
 
+// eslint-disable-next-line react/display-name
+const DialogTransition = React.forwardRef<typeof Fade, FadeProps>(
+  // eslint-disable-next-line react-memo/require-memo
+  (props, ref) => <Fade ref={ref} timeout={250} {...props} />,
```

Comment:
Should we move the timeout to a const? Is it supposed to match transition duration below?

Reply:
Done.